eol.js

```diff
@@ -17,9 +17,13 @@
   }
 
   function converts(to) {
-    return function(text) {
+    var f = function(text) {
       return text.replace(newline, to)
     }
+    f.toString = function() {
+      return to
+    }
+    return f
```

Comment:
- Let's use a function declaration for slightly nicer for debugging
- Let's rename `f` to `convert` to maybe make it a bit more readable
```
function converts(to) {
  function convert(text) {
    return text.replace(newline, to)
  }
  convert.toString = function() {
    return to
  }
  return convert
}
```